R-Tutorial/scripts/Installing_R.md

```diff
@@ -0,0 +1,17 @@
+# The website for downloading the R binary code is:
+
+[R-CRAN-PROJECT-DOWNLOAD](https://cloud.r-project.org/)
+
+# Once downloaded, the following steps are followed :
```

Comment:
Remove the space before the colon, like in first line (L#1) here, and please make that change to all the header lines (starting with # in md) to make the headings consistent. (we usually don't give a space before a colon)

Also, slightly smaller headings would be better (### or ####), but that's just my personal preference.